packages/web3-test/src/test-wallet.ts

```diff
@@ -96,8 +96,10 @@ export async function getSigner(alphAmount = ONE_ALPH * 100n, group = 0): Promis
     }
     const rootWallet = new PrivateKeyWallet({ privateKey: testPrivateKey })
     const wallet = PrivateKeyWallet.Random(group)
-    const destinations = [{ address: wallet.address, attoAlphAmount: alphAmount }]
-    await rootWallet.signAndSubmitTransferTx({ signerAddress: testAddress, destinations })
+    if (alphAmount > 0n) {
+      const destinations = [{ address: wallet.address, attoAlphAmount: alphAmount }]
+      await rootWallet.signAndSubmitTransferTx({ signerAddress: testAddress, destinations })
+    }
     return wallet
```

Comment:
I cannot make sense of it as a newcomer, we sign the transfer with `rootWallet` and return `PrivateKeyWallet.Random(group)` which seems to not hold any important information :thinking:

Reply:
I was writing similar tests last week while using this `getSigner()` method and the behavior was expected only when I returned and signed the transactions with the `rootWallet` :thinking: